### ISSUE TITLE
web ui table  long data display miss precision

### ISSUE THREAD

**Issue by yongchongwu** [apache/doris#10313]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

1.0.0

### What's Wrong?

web ui table  long data display miss precision
![image](https://user-images.githubusercontent.com/9245624/174789264-e4877d27-ac61-44f7-a1d8-f20c086d91b4.png)

### What You Expected?

web ui long data display right

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by zxealous** [comment]
There may be a problem with the front-end parsing the json data, the data returned by FE is no problem.

**Comment by zxealous** [comment]
I did a test, maybe there is a problem with the front-end parsing the data, the data returned by FE is correct.
![image](https://user-images.githubusercontent.com/54385456/174975940-35f4dd05-d9a4-43ae-b119-7772b3b8b484.png)
![image](https://user-images.githubusercontent.com/54385456/174976013-e5ea8528-d8ef-4da2-b662-9ebf4b43d5c0.png)

**Comment by yongchongwu** [comment]
Yes indeed ， this  is a problem with the front-end parsing the data,hope to deal with it
